types/k6/test/browser.ts

```diff
@@ -1203,105 +1203,105 @@ async function test() {
     //
     // Request
     //
-    const request = page.goto(url).then(r => r?.request());
+    const request = await page.goto(url).then(r => r?.request()!);
 
-    // $ExpectType Promise<Record<string, string> | undefined>
-    request.then(r => r?.allHeaders());
```

Comment:
This pattern is what had me believe that the APIs were already async. I updated it to improve readability and to avoid errors where returning a Promise inside another Promise would flatten into a Promise.